code/game/objects/items/devices/radio/headset.dm

```diff
@@ -1,4 +1,6 @@
 // Used for translating channels to tokens on examination
+//SPLURT EDIT REMOVAL BEGIN - MAPPING - (Moved to modular_splurt/code/game/objects/structures/globallist.dm)
+/*
```

Comment:
It's better to just do a nonmodular edit here rather than commenting the whole thing